example/lib/main.dart

```diff
@@ -73,10 +72,11 @@ Pagination involves a sequence of screens the user navigates sequentially. We ch
           ),
         ),
         pageTitle: Padding(
-          padding: const EdgeInsets.all(8.0),
+          padding: const EdgeInsets.symmetric(horizontal: 16),
           child: Text(
             'Material Colors',
-            style: Theme.of(context).textTheme.headlineMedium!.copyWith(fontWeight: FontWeight.bold),
+            style:
+                Theme.of(context).textTheme.headlineMedium!.copyWith(fontWeight: FontWeight.bold),
```

Comment:
`Theme.of(context).textTheme` is a good exmaple to cache in a variable, there is another text theme usage in this class.

`final TextTheme textTheme = Theme.of(context).textTheme;`

then call `textTheme.xxx` in multiple places

Reply:
done in this: 1910f40794a1e00e56fbbbf74b020ce827e32e7d